README.md

```diff
@@ -34,6 +34,7 @@ To install PyTorch/XLA nightly build in a new TPU VM:
 ```
 pip3 install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
Regarding https://github.com/pytorch/xla/issues/7697#issuecomment-2251576498, can you also add an optional command at the end to install compatible `torchaudio`, `torchtext`, and `torchvision` from the PyTorch nightly index?

Reply:
we rarely ever need torchaudio or torchtext. I think we put torchvision here because I use resnet in my examples.

Reply:
I'll follow up on this since it's tangential to this PR's topic. IMO it's still useful to include the other official ecosystem packages. It's very easy to break nightly `torch_xla` by installing a stable `torchtext` that pulls in an incompatible stable `torch`. We've made this mistake numerous times in our own test cases.

Reply:
If we want to install `torchaudio`, `torchtext`, etc. from the PyTorch nightly index, the command from PyTorch index page https://screenshot.googleplex.com/8xKE2QXabRsqq9D is `pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu`. They should be compatible already.